caasp-ingress-nginx-controller-image/caasp-ingress-nginx-controller-image.kiwi

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- OBS-AddTag: caasp/v5/ingress-nginx-controller:%%PKG_VERSION%%-rev<VERSION> caasp/v5/ingress-nginx-controller:%%PKG_VERSION%%-rev<VERSION>-build<RELEASE> caasp/v5/ingress-nginx-controller:beta -->
+<image xmlns:suse_label_helper="com.suse.label_helper" schemaversion="6.9" name="ingress-nginx-controller">
```

Comment:
Replace `ingress-nginx-controller` with `nginx-ingress-controller` similar to the name of the package submitted.

Reply:
see https://github.com/SUSE/caasp-container-images/pull/105#discussion_r461407031